pkgs/development/python-modules/autobahn/default.nix

```diff
@@ -24,11 +37,12 @@ buildPythonPackage rec {
   # Tests do no seem to be compatible yet with pytest 5.1
   # https://github.com/crossbario/autobahn-python/issues/1235
```

Comment:
This is is closed. Do the tests work now?

Reply:
Partially at least they do.